### ISSUE TITLE
--fetch doesn't work

### ISSUE THREAD

**Issue by jpopelka** [fedora-ruby/gem2rpm#55]
While `gem fetch github-linguist && gem2rpm github-linguist-4.7.5.gem` works OK,
any of

- gem2rpm --fetch github-linguist
- gem2rpm --fetch github-linguist-4.7.5
- gem2rpm --fetch github-linguist-4.7.5.gem

throw
```
/usr/share/ruby/open-uri.rb:224:in `open_loop': redirection forbidden: http://rubygems.org/api/v1/gems/github-linguist-4.7.5.json -> https://rubygems.org/api/v1/gems/github-linguist-4.7.5.json (RuntimeError)
	from /usr/share/ruby/open-uri.rb:150:in `open_uri'
	from /usr/share/ruby/open-uri.rb:716:in `open'
	from /usr/share/ruby/open-uri.rb:34:in `open'
	from /usr/share/gems/gems/gem2rpm-0.11.2/bin/gem2rpm:103:in `<top (required)>'
	from /usr/bin/gem2rpm:23:in `load'
	from /usr/bin/gem2rpm:23:in `<main>'
```
when using rubygem-gem2rpm-0.11.2-1.fc23.noarch or the following when I `gem install gem2rpm`
```
/usr/share/ruby/open-uri.rb:224:in `open_loop': redirection forbidden: http://rubygems.org/api/v1/gems/github-linguist-4.7.5.json -> https://rubygems.org/api/v1/gems/github-linguist-4.7.5.json (RuntimeError)
	from /usr/share/ruby/open-uri.rb:150:in `open_uri'
	from /usr/share/ruby/open-uri.rb:716:in `open'
	from /usr/share/ruby/open-uri.rb:34:in `open'
	from /home/jpopelka/.gem/ruby/gems/gem2rpm-0.11.2/bin/gem2rpm:103:in `<top (required)>'
	from /home/jpopelka/bin/gem2rpm:23:in `load'
	from /home/jpopelka/bin/gem2rpm:23:in `<main>'
```

**Comment by voxik** [comment]
This is actually fixed by #52.

**Comment by voxik** [comment]
Precisely, ```gem2rpm --fetch github-linguist``` works now and the other cases should be improved a bit by #66
